<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for fastapi-errors</title>
  </head>
  <body>
    <h1>Links for fastapi-errors</h1>
<a href="https://files.pythonhosted.org/packages/21/c5/8a2823a6806113eab21671b1df6fde8d4e9738a84d679bfdeed0835af06d/fastapi_errors-0.0.1-py2.py3-none-any.whl#sha256=cb5de0d5c3b8fff7ce8ae787c6901facebe403ce55d80230996bde5c3d44927c" data-dist-info-metadata="sha256=7783774ab15f1f156eec71db7df2e7daf4883179e244c4ca3ff3fed92a703ee8" data-core-metadata="sha256=7783774ab15f1f156eec71db7df2e7daf4883179e244c4ca3ff3fed92a703ee8">fastapi_errors-0.0.1-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/3b/03/1e76a0dd5c46b72748771ff7f05cf43e7a90e9689357bd3285409c9b5ba9/fastapi_errors-0.0.1.tar.gz#sha256=adde4e6977909d1eeffe320d8f3a430428a0c5e525ea580e130a3ac427baef59" >fastapi_errors-0.0.1.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/e7/7c/290ab5b3c4725f8ee0bb121924436a343b44a891be11063f1dbe405d29a5/fastapi_errors-0.0.2-py2.py3-none-any.whl#sha256=e22c84dda8d4de74ea2a0cd95238f12c6165151054067ea61241bae6223f2260" data-dist-info-metadata="sha256=0e47f09a5f29e8d5c0edb9fdae31b54c278ec011bc2feaecc0344b0fb015d950" data-core-metadata="sha256=0e47f09a5f29e8d5c0edb9fdae31b54c278ec011bc2feaecc0344b0fb015d950">fastapi_errors-0.0.2-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/74/9c/d988f66b968a48a0537395e77d8b8fe7937aa4fc0c66c1eeb3faa13c2a8d/fastapi_errors-0.0.2.tar.gz#sha256=998fc2ba1865b6c773a9816da99aa4592a928374df4a99aa7b0ea3b251b251f9" >fastapi_errors-0.0.2.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/ac/30/1e1a8d4edf7df08c6837c1cf8de43cb8757744098d5efb92a1ce7921f0e1/fastapi_errors-0.0.4-py2.py3-none-any.whl#sha256=ae2e8cefbf1e4e3a57466656a8078db20aac349e1f2dbb8353ab40875ef7c9e0" data-dist-info-metadata="sha256=4852e03cc8f54262ffcc79078934c33fdbe59b530855cdc4da6475d21ecb3f04" data-core-metadata="sha256=4852e03cc8f54262ffcc79078934c33fdbe59b530855cdc4da6475d21ecb3f04">fastapi_errors-0.0.4-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/58/c8/45b00a605a0d67a5ad9b68c443c1ab9536b2f6c186357ad1542b45a27058/fastapi_errors-0.0.4.tar.gz#sha256=d4144999c5bfc6c0a7271952e4663a994a2cddfa2c8bc6d22f0d44647289e326" >fastapi_errors-0.0.4.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/72/43/5a7adddeb453475045074207af717676c20139bb0c0dc0ac172d6032fa4b/fastapi_errors-0.0.5-py2.py3-none-any.whl#sha256=f4d0f1725841d6c429815b17978410a97004d864864f7b25afb6a85cee484679" data-dist-info-metadata="sha256=cad3c0d10d6a671ca608969e72153eb8aca82a80e8ad20c79ee635cda940f345" data-core-metadata="sha256=cad3c0d10d6a671ca608969e72153eb8aca82a80e8ad20c79ee635cda940f345">fastapi_errors-0.0.5-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/c7/f9/5e6236845ab0fac626e6c9d3d24df38b59c51a1614185456c6029fc7f9c7/fastapi_errors-0.0.5.tar.gz#sha256=7fa312aff17b73d794ff9f9ce6080f6ffeadaf7174dc01276df23cf70e70eb87" >fastapi_errors-0.0.5.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/1a/40/573c27e89767c1622f3fe5fa039850572b078846602502fb41a716d68dfb/fastapi_errors-0.0.6-py2.py3-none-any.whl#sha256=a7d8ed19da6f4de99b8bc1bc37add8d5f9ab6f9335288938370a49fea86b4631" data-dist-info-metadata="sha256=b349d55dfcf8a5de96398cf145a2922f83e459246c475696ed8590cb04d2cd9d" data-core-metadata="sha256=b349d55dfcf8a5de96398cf145a2922f83e459246c475696ed8590cb04d2cd9d">fastapi_errors-0.0.6-py2.py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/97/97/0a51ba60bacdf8dce3239aded45abeaf9ad849a05dad27c337b629d83096/fastapi_errors-0.0.6.tar.gz#sha256=ad29a51e811087eb760a2858b49eefa45a03f2171d8cc608967fe5841e66e794" >fastapi_errors-0.0.6.tar.gz</a><br />
</body>
</html>
<!--SERIAL 23489112-->